packages/chevrotain/src/parse/grammar/lookahead.ts

```diff
@@ -190,10 +193,10 @@ export function buildAlternativesLookAheadFunc(
       singleTokenAlts,
       (result, currAlt, idx) => {
         forEach(currAlt, (currTokType) => {
-          if (!has(result, currTokType.tokenTypeIdx)) {
-            result[currTokType.tokenTypeIdx] = idx
+          if (!has(result, currTokType.tokenTypeIdx!)) {
```

Comment:
I think we need `!` due to issues described in: #1041

Reply:
fixing this with a workaround (another type with `Required<>` mapping will require too many changes.
Too bad `tsc` does not complain about unneeded `not null` asseriton.
It may be difficult to spot those once #1041 will be fixed.
Maybe there is an eslint rule for that.

Reply:
There is https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unnecessary-type-assertion.md